mpisppy/utils/wxbarreader.py

```diff
@@ -39,18 +39,42 @@
 n_proc = MPI.COMM_WORLD.Get_size()
 rank = MPI.COMM_WORLD.Get_rank()
 
+def add_options_to_config(cfg):
+
+    cfg.add_to_config("W_and_xbar_reader",
+                      description="Enables the w and xbar reader (default False)",
+                      domain=bool,
+                      default=False)
+    cfg.add_to_config("init_W_fname",
+                      description="Path of initial W file (default None)",
+                      domain=str,
+                      default=None)
+    cfg.add_to_config("init_Xbar_fname",
+                      description="Path of initial Xbar file (default None)",
+                      domain=str,
+                      default=None)
+    cfg.add_to_config("init_separate_W_files",
+                      description="If True, W is read from separate files (default False)",
+                      domain=bool,
+                      default=False)
+
 class WXBarReader(mpisppy.extensions.extension.Extension):
     """ Extension class for reading W values
     """
     def __init__(self, ph):
 
+        assert 'cfg' in ph.options
+        self.cfg = ph.options['cfg']
+        if self.cfg.get("W_and_xbar_reader") is None or not self.cfg.W_and_xbar_reader:
+            self.not_active = True
```

Comment:
Since this is new code, I'll just say I don't like negative `bool`s (i.e., the flag should be called `active`).

Not a reason to hold up this PR.

Reply:
I will make the change.